deploy/docs/existingPrometheusDoc.md

```diff
@@ -93,14 +93,26 @@ kubectl get configmap sumologic-configmap \
 kubectl apply --namespace=<destination-namespace> -f -
 ```
 
-Run the following commands to update the [remote write configuration](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) of the prometheus operator with the prometheus overrides we provide in our [prometheus-overrides.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/prometheus-overrides.yaml).
+Run the following commands to update the [remote write configuration](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write) of the prometheus operator with the prometheus overrides based on our `values.yaml`.
 
-Run the following command to download our prometheus-overrides.yaml file. Please review our configuration as it will be applied to your existing operator configuration.
+First, generate the Prometheus Operator `prometheus-overrides.yaml` by running commands below:
 
 ```bash
-curl -LJO https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/release-v1.3/deploy/helm/prometheus-overrides.yaml
+ # using kubectl
+ kubectl run tools \
+  -it --quiet --rm \
+  --restart=Never -n sumologic \
+  --image sumologic/kubernetes-tools:master \
+  -- template-dependency prometheus-operator > prometheus-overrides.yaml
```

Comment:
This requires a specific version of kubernetes-tools right? Should we explicitly state the image version we want customers to use in case it is cached?

Reply:
So the question is do we want to do it that way or point to the static file in repository as we do it now?

Reply:
I will update it when we release `sumologic-kubernetes-tools:2.0.0`

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1076